_pages/tourney-rules.html

```diff
@@ -6,117 +6,109 @@
+                <li>The Staff Team has several screen sharing tools at their disposal. <i>Not downloading them will be considered denying to screenshare</i>.</li>
+                <li>Any player found <code>editing/deleting</code> anything, having <code>cheats/autoclickers</code> or using any program that blocks our screen sharing tools will receive a <b>Tournament Ban</b>.</li>
+                <li>Intentionally delaying the screenshare may result in a <b>Punishment</b>.</li>
+                <li>Any <code>screenshare blockers</code> (firewalls, programs designed to mislead and block screenshare tools, etc.) will be considered an attempt to cloud the screenshare, and may be faced with a <b>Tournament Ban</b>.</li>
+            </ol>
+        </div>
+    </div>
+    <div class="pb-2 mt-4 mb-2 border-bottom">
+        <h2 class="page-heading">C. Infractions</h2>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <p><b>The infracions system works as follows:</b></p>
+            <ol>
+                <li><b>Warnings:</b> For any minor offence, the player will receive a <code>warning</code>. This is mainly due to toxicity, spamming or any minor block glitching that does not change the outcome of the match (e.g, block glitching to avoid death in mid). <i>A fourth warning will result in a <b>punishment</b> for that player</i>.</li>
+                <li><b>Punishments:</b> The player will miss the next map for the first <code>punishment</code> and the rest of the series for the second one. The Staff Team will be giving punishments for things like extreme toxicity (harassment, being extremely toxic, etc), stream sniping, block glitching and changing the outcome of a match (e.g. getting out of the wool while block glitching, getting into the wool block glitching, getting proximity that changes the final result while block glitching, etc.) or by receiving <code>3 warnings</code>. <i>A third punishment will result in a <b>Tournament Ban</b></i>.</li>
+                <li><b>Tournament Ban:</b> This is the worst type of punishment a player may get. A team can receive up to three <code>Tournament Bans</code> before being disqualified from the tournament. This is reserved for extreme cases such as hacking or receiving up to <code>3 punishments</code>. For the first two Tournament Bans, the player will be disqualified from the tournament and the team will be given a loss for every map that player has played during that series. For the third one, the team will be disqualified from the tournament. <i>All players being issued a Tournament Ban will be <b>blacklisted</b> from future tournaments</i>.</li>
+            </ol>
+        </div>
+    </div>
+    <div class="pb-2 mt-4 mb-2 border-bottom">
+        <h2 class="page-heading">D. Blacklist</h2>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <ul>
```

Comment:
```suggestion
            <ol>
```
I think this should be an ordered list like all the other rules on this page and the rules page.